firebase-sessions/src/main/kotlin/com/google/firebase/sessions/settings/LocalOverrideSettings.kt

```diff
@@ -18,32 +18,21 @@ package com.google.firebase.sessions.settings
 
 import android.content.Context
 import android.content.pm.PackageManager
-import android.os.Build
 import android.os.Bundle
 import kotlin.time.Duration
 import kotlin.time.DurationUnit
 import kotlin.time.toDuration
 
 internal class LocalOverrideSettings(context: Context) : SettingsProvider {
+  @Suppress("DEPRECATION") // TODO(mrober): Use ApplicationInfoFlags when target sdk set to 33
```

Comment:
With the version based checks removed, do we know if the new version APIs are backward compatible?

Reply:
They are, I tested it. And Firebase will be setting the target sdk to 33 soon anyway, then we can remove this check and use the new api only.